.github/workflows/autodoc_cicd.yml

```diff
@@ -98,34 +98,34 @@ jobs:
         uses: actions/cache@v3
         with:
           path: ~/.cache/pip
-          key: Python-pydita-generatedcommands-v${{ env.RESET_PIP_CACHE_2 }}-${{ inputs.MAIN_PYTHON_VERSION }}-${{ hashFiles('pyproject.toml') }}
+          key: Python-pyconverter-generatedcommands-v${{ env.RESET_PIP_CACHE_2 }}-${{ inputs.MAIN_PYTHON_VERSION }}-${{ hashFiles('pyproject.toml') }}
           restore-keys: |
-            Python-pydita-generatedcommands-v${{ env.RESET_PIP_CACHE_2 }}-${{ inputs.MAIN_PYTHON_VERSION }}
+            Python-pyconverter-generatedcommands-v${{ env.RESET_PIP_CACHE_2 }}-${{ inputs.MAIN_PYTHON_VERSION }}
       
-      - name: "Install pydita-generatedcommands"
+      - name: "Install pyconverter-generatedcommands"
         run: |
           pip install .
       
-      - name: "Retrieve PyDita-Generatedcommands version"
+      - name: "Retrieve PuConverter-Generatedcommands version"
```

Comment:
```suggestion
      - name: "Retrieve PyConverter-Generatedcommands version"
```

Reply:
Good catch! Thank you @Revathyvenugopal162